book/src/user/supported-platforms.md

```diff
@@ -0,0 +1,48 @@
+# Platform Support
+
+Support for different platforms ("targets") are organized into three tiers,
+each with a different set of guarantees. For more information on the policies
+for targets at each tier, see the [Target Tier Policy](target-tier-policy.md).
+
+Targets are identified by their "target triple", which is the string to inform
+the compiler what kind of output should be produced.
+
+## Tier 1
+
+Tier 1 targets can be thought of as "guaranteed to work". The Zebra project
+builds official binary releases for each tier 1 target, and automated testing
+ensures that each tier 1 target builds and passes tests after each change.
+
+For the full requirements, see [Tier 1 target policy](target-tier-policy.md#tier-1-target-policy) in the Target Tier Policy.
+
+target | notes
+-------|-------
+`x86_64-unknown-linux-gnu` | 64-bit Linux (kernel 4.19+, glibc 2.33+)
```

Comment:
Where did these kernel and glibc versions come from?
Are they from our GitHub or Docker runners?

(I have similar questions about the other tiers.)

Reply:
Both, but I'll make this more clear with the specific OS.

I think we should use Rust version instead of glibc version